common/test/acceptance/tests/studio/test_studio_course_create.py

```diff
@@ -144,3 +146,66 @@ def test_create_course_with_existing_org_via_autocomplete(self):
         self.assertTrue(self.dashboard_page.has_course(
             org=new_org, number=self.course_number, run=self.course_run
         ))
+
+    def test_error_appears_with_long_tuple(self):
+        """
+        Scenario: Ensure that the course creation is not successful with 66 characters long tuple.
+        Given I have filled course creation from with combined length of 66 characters for Organization, course Number
+        and course Run.
+        And I have a valid course name
+        Then form validation should not pass
+        And I see error for combined length longer than 65
+        """
+        course_org = "012345678901234567890123456789"
+        course_number = ''.join(random.choice(string.digits) for _ in range(30))
+        course_run = "0123456"
+        self.auth_page.visit()
+        self.dashboard_page.visit()
+        self.assertTrue(self.dashboard_page.new_course_button.present)
```

Comment:
Note for future reference: pytest has [special handling](https://docs.pytest.org/en/latest/assert.html#assert) of normal `assert` statements to produce nice error messages, so instead of `self.assertTrue(condition)` you could just use `assert condition`.